src/osd/PrimaryLogPG.cc

```diff
@@ -3539,7 +3539,7 @@ bool PrimaryLogPG::inc_refcount_by_set(OpContext* ctx, object_manifest_t& set_ch
     refs);
   bool need_inc_ref = false;
   if (!refs.is_empty()) {
-    ManifestOpRef mop = std::make_shared<ManifestOp>(new RefCountCallback(ctx, osd_op));
+    ManifestOpRef mop(std::make_shared<ManifestOp>(nullptr));
```

Comment:
Give ManifestOp a default constructor and use it here.  Remove this one if there are no other users.